proposals/0007-access-policy.md

```diff
@@ -0,0 +1,126 @@
+---
+simd: '0007'
+title: SIMD Access Policy
+authors:
+- Jacob Creech (Solana Foundation)
+category: Meta
+type: Meta
+status: Draft
+created: 2022-12-10
+---
+
+## Summary
+
+3 levels of access, in the order of increasing access:
+
+1. Triage
+
+    Requirement: One voucher from anyone with level 2 or above access.
+
+2. Core
+
+    Requirement: One voucher from anyone with level 3 access or two from
+    anyone with level 2 access.
+
+3. Maintain
+
+    Requirement: One voucher from anyone with level 3 access. This permission
+    is usually reserved for those maintaining the SIMD process.
```

Comment:
What's the process for removing problematic users? I feel that one user with Maintain access could remove a Core or Triage user. Maintainers that want to remove other maintainers must cage match

Reply:
Added some clarification around revoking access. LMK what you think. No cage matches unfortunately.